src/layout/layout.js

```diff
@@ -190,27 +195,39 @@
    *
    * @private
    */
-  MaterialLayout.prototype.screenSizeHandler_ = function() {
-    if (this.screenSizeMediaQuery_.matches) {
-      this.element_.classList.add(this.CssClasses_.IS_SMALL_SCREEN);
 
```

Comment:
You need jsdoc for screenSizeHandler.

Reply:
Your right the JSDoc is not accurate anymore